src/hotspot/share/opto/vectornode.hpp

```diff
@@ -262,18 +262,23 @@ class AddReductionVLNode : public ReductionNode {
 // Vector add float as a reduction
 class AddReductionVFNode : public ReductionNode {
 private:
-  bool _requires_strict_order; // false in Vector API.
+  // True if add reduction operation for floats requires strict ordering.
+  // The value is true when add reduction for floats is auto-vectorized as auto-vectorization
+  // mandates strict ordering but the value is false when this node is generated through VectorAPI
+  // as VectorAPI does not impose any such rules on ordering.
```

Comment:
The comment can be more better. But I leave it to a reviewer who is proficient in english to help you improve it.